spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py

```diff
@@ -220,6 +234,11 @@ def _transform(self, df: DataFrame) -> DataFrame:
                         )
                     # get resource name
                     result_df: DataFrame = df.sql_ctx.table(view)
+                    need_sorting: bool = (
+                        True
+                        if (sort_data and view in sort_data.get("views"))  # type: ignore
+                        else False
```

Comment:
what are we doing with views?

Reply:
we need to sort the data only for the eob views.